### PR TITLE
[MPDX-8440] Remove TNT import file upload restriction

### PR DESCRIPTION
**Pull request by canac**
## Description

The Amplify maximum body size for lambda file uploads of 1MB is too small for some users' multi-MB TNT uploads. Instead of uploading TNT exports to the lambda, upload them directly to the API and remove the file size restriction.

I also cleaned up a few things in the `TntConnect` component:
* Give variable a more descriptive name
* Remove a redundant call to revokeObjectURL

## Testing

I downloaded and used [this fixture](https://github.com/CruGlobal/mpdx_api/blob/master/spec/fixtures/tnt/tnt_export.xml) to test the upload.

[MPDX-8440](https://jira.cru.org/browse/MPDX-8440)

https://secure.helpscout.net/conversation/2756145432/1255279/
https://secure.helpscout.net/conversation/2756616046/1255472/

## Checklist:

- [x] I have given my PR a title with the format "MPDX-(JIRA#) (summary sentence max 80 chars)"
- [x] I have applied the appropriate labels. (_Add the label "On Staging" to get the branch automatically merged into staging._)
- [x] I have requested a review from another person on the project

### CODE REVIEW

File: src/components/Tool/TntConnect/uploads/uploadTntConnect.ts [hunk 1]
```diff
@@ -45,19 +38,24 @@ export const uploadTnt = async ({
...
+  const res = await fetch(
+    `${process.env.REST_API_URL}account_lists/${accountListId}/imports/tnt`,
+    {
+      method: 'POST',
+      headers: {
+        authorization: `Bearer ${apiToken}`,
+      },
+      body: form,
+    },
+  ).catch(() => {
```
Comment (canac) [inline src/components/Tool/TntConnect/uploads/uploadTntConnect.ts:55]: `fetch` throws if there was a network error such that it couldn't make the request it all. It resolves to a response with `ok: false` if there was an HTTP error (i.e. 4xx or 5xx status code). That's why I switched this error message to be more accurate.

File: src/components/Tool/TntConnect/TntConnect.tsx [hunk 2]
```diff
@@ -174,10 +178,6 @@ const TntConnect: React.FC<Props> = ({ accountListId }: Props) => {
       return;
     }
 
-    if (tntFile) {
```
Comment (canac) [inline src/components/Tool/TntConnect/TntConnect.tsx:177]: The old blob will already be released by the `useEffect` cleanup function on line 164.

**Review by dr-bizz — COMMENTED** [review]
The code looks great! As I didn't build it, I will let Caleb C test the UI as he built the UI and had a lot of the TNT files.

**Comment by dr-bizz** [comment]
@caleballdrin Can you test the UI of this PR by uploading some of your test TNT files?

**Review by caleballdrin — APPROVED** [review]
I just tested it and it worked! However, I didn't try it with a file over 1mb.

**Comment by canac** [comment]
It worked with the 13.5MB one in the HelpScout ticket https://secure.helpscout.net/api/v1/conversations/2756145432/attachments/733246685/file

**Pull request merged**